client/my-sites/plans/jetpack-plans/use-get-plans-grid-products.ts

```diff
@@ -78,14 +84,18 @@ const useSelectorPageProducts = ( siteId: number | null ): PlanGridProducts => {
 
 	// If Jetpack Anti-spam is directly or indirectly owned, continue, otherwise make it available.
 	if (
-		! ownedProducts.some( ( ownedProduct ) => JETPACK_ANTI_SPAM_PRODUCTS.includes( ownedProduct ) )
+		! ownedProducts.some( ( ownedProduct ) =>
+			( JETPACK_ANTI_SPAM_PRODUCTS as ReadonlyArray< string > ).includes( ownedProduct )
+		)
 	) {
 		availableProducts = [ ...availableProducts, ...JETPACK_ANTI_SPAM_PRODUCTS ];
 	}
 
 	// If Jetpack VideoPress is directly or indirectly owned, continue, otherwise make it available.
 	if (
-		! ownedProducts.some( ( ownedProduct ) => JETPACK_VIDEOPRESS_PRODUCTS.includes( ownedProduct ) )
+		! ownedProducts.some( ( ownedProduct ) =>
+			( JETPACK_VIDEOPRESS_PRODUCTS as ReadonlyArray< string > ).includes( ownedProduct )
+		)
```

Comment:
these are all abstraction leaks, by the way, which is evidenced by the fact that your type-only PR requires so much effort to change them